Workbooks/SapMonitor/SapHanaInfrastructure/SapHanaInfrastructure.workbook

```diff
@@ -0,0 +1,376 @@
+        "size": 0,
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "crossComponentResources": [
+          "{LogAnalyticsWorkspaceArmId}"
+        ],
+        "visualization": "linechart",
+        "graphSettings": {
+          "type": 0
+        }
+      },
+      "name": "query - 7 - Copy - Copy"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "CPU Anomlies"
+      },
+      "name": "text - 9"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "let startDate = startofday(ago(7d)); // go back in time nn days\r\nlet endDate = now(); // what is the date now\r\nlet projectTo = now()+7d; // project forward nn days\r\nlet projectForward = 7; // must be same as projectTo value\r\nSapHana_LoadHistory_CL\r\n|where TimeGenerated > ago(7d)\r\n| make-series avgCPU=avg(CPU_d) default=0 on TimeGenerated in range(ago(7d),projectTo, 1h) by HOST_s\r\n|extend (flag, Anomalyscore,baseline) = series_decompose_anomalies(avgCPU)\r\n|extend anomalies = series_multiply(5,Anomalyscore)\r\n|project-away baseline, flag, Anomalyscore",
+        "size": 0,
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "crossComponentResources": [
+          "{LogAnalyticsWorkspaceArmId}"
+        ],
+        "visualization": "linechart"
+      },
+      "name": "query - 10"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "Memory Anomalies"
+      },
+      "name": "text - 16"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "let startDate = startofday(ago(7d)); // go back in time nn days\r\nlet endDate = now(); // what is the date now\r\nlet projectTo = now()+7d; // project forward nn days\r\nlet projectForward = 7; // must be same as projectTo value\r\nSapHana_LoadHistory_CL\r\n|where TimeGenerated > ago(7d)\r\n| make-series avgMEM=avg(MEMORY_USED_d/MEMORY_SIZE_d)*100 default=0 on UTC_TIMESTAMP_t in range(ago(7d),projectTo, 1h) by HOST_s\r\n|extend (flag, Anomalyscore,baseline) = series_decompose_anomalies(avgMEM)\r\n|extend anomalies = series_multiply(5,Anomalyscore)\r\n|project-away baseline, flag, Anomalyscore",
+        "size": 0,
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "crossComponentResources": [
+          "{LogAnalyticsWorkspaceArmId}"
+        ],
+        "visualization": "linechart"
+      },
+      "name": "query - 10 - Copy"
+    }
+  ],
+  "fallbackResourceIds": [
```

Comment:
the resources down here are the thing the PR is rejecting.  in a template, you can remove this whole section